dxr/build.py

```diff
@@ -447,7 +447,7 @@ def index_file(tree, tree_indexers, path, es, index):
     # will not accept the path, so just move on.
     rel_path = relpath(path, tree.source_folder)
     try:
-        rel_path = rel_path.decode('utf-8')
+        rel_path = unicode(rel_path, 'utf-8')
```

Comment:
The paths arrive from the os.walk in unignored, so I thought that's the earliest point I can convert them.